rethinkdb/rethinkdb.d.ts

```diff
@@ -49,19 +79,61 @@ declare module "rethinkdb" {
         close(): Promise<void>;
     }
 
+    interface Row extends Expression<any> {
+      (name: string): Expression<any>;
+    }
+
+    /**
+     * Connection options.
+     *
+     * See: https://rethinkdb.com/api/javascript/connect/
+     */
     interface ConnectionOptions {
-        host: string;
-        port: number;
+        /** The host to connect to (default `localhost`) */
+        host?: string;
+
+        /** The port to connect on (default `28015`) */
+        port?: number;
+
+        /** The default database (default `test`) */
         db?: string;
-        authKey?: string;
```

Comment:
I think it might be documentation issue... out of 4 drivers, only Java has the same "mistake" about the authKey... the list of accepted options doesn't have authKey in any of the drivers. I'm inclined to "trust" that more. Having said that, I don't feel very strongly about this and can remove this change.

![reql command reference - rethinkdb_20161107161527](https://cloud.githubusercontent.com/assets/65633/20081639/c68f4b30-a505-11e6-905d-5e3f910dd9ae.png)